src/core/componentSystems/TextSystem.js

```diff
@@ -81,7 +75,11 @@ export class TextSystem extends MRSystem {
      * Relies on the parent's implementation. (see [MRSystem.needsSystemUpdate](https://docs.mrjs.io/javascript-api/#mrsystem.needssystemupdate) for default).
      */
     set needsSystemUpdate(bool) {
-        super.needsSystemUpdate = bool;
+        console.log('Updating a value that does not need updating. TextSystem`s `needsSystemUpdate` solely depends on registry size \
+            instead of a boolean. This is required s.t. we can monitor and properly update the system when the text content has changed \
```

Comment:
is this log necessary?

Reply:
yes and no - yes to inform the user that this function technically does nothing though the function is needed to be implemented - no in that i can make the log a lot shorter and keep most of it as a code comment.

i can switch it to the code comment + small message version - one sec